### PR TITLE
CLI

### PR DESCRIPTION
**Pull request by liorblob**

### CODE REVIEW

File: demisto_sdk/core.py [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-from .packages.splitter import Splitter
+
+from .yaml_tools.unifier import Unifier
+from .yaml_tools.extractor import Extractor
 from .common.configuration import Configuration
 
 
-class Client:
+class DemistoSDK:
+    SCRIPT = 'script'
```
Comment (Itay4) [inline demisto_sdk/core.py]: lets add constants file and define these there

File: demisto_sdk/main.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                                choices=[True, False], type=str2bool, default=None)
+
+    unify_parser.add_argument("-i", "--indir", help="The path to the files to unify", required=True)
+    unify_parser.add_argument("-o", "--outdir", help="The output dir to write the unified yml to", required=True)
+
+    args = parser.parse_args()
+
+    sdk = DemistoSDK()
+
+    if args.command == 'extract':
```
Comment (Itay4) [inline demisto_sdk/main.py]: is this only temp?

Comment (liorblob) [reply demisto_sdk/main.py]: it's the general direction.. why?

Comment (Itay4) [reply demisto_sdk/main.py]: can see how this quickly become messy, can we generlize this ?

**Pull request merged**